src/keeper/contracts/conditions/NFTs/TransferNFTCondition.ts

```diff
@@ -87,6 +91,13 @@ export class TransferNFTCondition extends ProviderCondition<TransferNFTCondition
         }
     }
 
+    public nftContractFromService(service: ServiceCommon): string {
+        const holder = findServiceConditionByName(service, 'transferNFT')
+        if (!holder) throw new Error('TransferNFT condition not found!')
+        const res = holder.parameters.find(p => p.name === '_contractAddress').value as string
```

Comment:
It could be undefined
```suggestion
        const res = holder.parameters.find(p => p.name === '_contractAddress')?.value as string
```